### DISCUSSION TITLE
Welcome to GTN Discussions!

### DISCUSSION THREAD

**Discussion by shiltemann** [galaxyproject/training-material#2196] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by annvuyanzi** [comment]
I am Ann .looking forward to being an outreachy intern ,be part of this GTN community to learn, work  and share knowledge.

**Comment by bebatut** [comment]
Welcome !!
Thanks @shiltemann for starting that

**Comment by garimavs** [comment]
Hello everyone!
This is Garima Singh, and I'm an Outreachy Applicant in Galaxy Community. I have so far worked mainly with the FAQs in the GTN and HUB.

I have spotted some issues in the GTN FAQs, which I have mentioned in point no: #3455 #3442 #3440 #3439(resolved)
I want to work on these (if they are an issue to be considered), and I would also appreciate it if I could get some guidance to get started with these.

**Reply by shiltemann** [reply]
Awesome! I will try to respond on those issues today or tomorrow with some tips to get you started (am teaching this week so don't have a lot of time, but should be able to help you in between sessions)

**Comment by garimavs** [comment]


**Reply by hexylena** [reply]
Discussions generally isn't the right place for this, please file these as issues in the future.

This is because the offset is slightly different, when you click on it, it jumps to the heading, but the TOC expects you to be a few pixels lower. I'm not sure if we can easily fix this, we can't make it jump somewhere different, and the TOC plugin isn't something I'd be keen to modify (it's maintained by someone else)

**Reply by hexylena** [reply]
ah you already filed it here https://github.com/galaxyproject/training-material/issues/3442

**Reply by garimavs** [reply]
Oh, thanks for mentioning @hexylena, and sorry for the trouble. I will delete this comment and directly work on the issue itself.

**Reply by hexylena** [reply]
No worries! Just for the next time :)
